### PR TITLE
Use o-labels indicators and timestamps

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.1.2 to 50c04f117e1d208cda448b9c76e7d4a8bbbc688e
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
👋 Percy has finished running the visual regression testing!

**Comment by notlee** [comment]
ah, I forgot we're out of percy this month

**Comment by notlee** [comment]
Note that this intentionally makes `o-teaser__timestamp--inprogress` animate (requested by design, discussed in the ft-uxd slack channel)

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.1.2 to 8b573a5570f6ad9aaae1f07bf5026a17dab52583
css: 0.36kb increase (0.09kb/gzip)

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.1.2 to 50c48cdad7e4197feb5dde972f75aa10b5e8084c
css: 0.35kb increase (0.08kb/gzip)

File: demos/src/demo-syndi.mustache [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			{{/article-timestamp}}
 
 			{{#article-headshot}}
-				<img src="{{article-headshot}}" alt=" " class="o-teaser__headshot">
+				<img src="{{article-headshot}}" alt="exmaple article headshot image" class="o-teaser__headshot">
```
Comment (JakeChampion) [inline demos/src/demo-syndi.mustache]:
```suggestion
				<img src="{{article-headshot}}" alt="example article headshot image" class="o-teaser__headshot">
```

File: demos/src/demo.mustache [hunk 2]
```diff
@@ -38,7 +38,7 @@
 			{{/article-timestamp}}
 
 			{{#article-headshot}}
-				<img src="{{article-headshot}}" alt=" " class="o-teaser__headshot">
+				<img src="{{article-headshot}}" alt="exmaple article headshot image" class="o-teaser__headshot">
```
Comment (JakeChampion) [inline demos/src/demo.mustache]:
```suggestion
				<img src="{{article-headshot}}" alt="example article headshot image" class="o-teaser__headshot">
```

File: demos/src/pa11y.mustache [hunk 3]
```diff
@@ -106,7 +106,7 @@
 			{{/article-timestamp}}
 
 			{{#article-headshot}}
-				<img src="{{article-headshot}}" alt=" " class="o-teaser__headshot">
+				<img src="{{article-headshot}}" alt="exmaple article headshot image" class="o-teaser__headshot">
```
Comment (JakeChampion) [inline demos/src/pa11y.mustache]:
```suggestion
				<img src="{{article-headshot}}" alt="example article headshot image" class="o-teaser__headshot">
```

File: demos/src/pa11y.mustache [hunk 4]
```diff
@@ -449,7 +449,7 @@
...
```
Comment (JakeChampion) [inline demos/src/pa11y.mustache]:
```suggestion
				<img src="{{article-headshot}}" alt="example article headshot image" class="o-teaser__headshot">
```

**Review by JakeChampion — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v5.1.3 :tada:

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.1.2 to 5f1f5446246cdd5cda7745bf8c76d35236659282
css: 0.35kb increase (0.08kb/gzip)

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 5.1.2 to efa9260d1a9870671dc5ab2cefb49932fea632e4
css: 0.35kb increase (0.08kb/gzip)

**Pull request merged**